.pre-commit-config.yaml

```diff
@@ -0,0 +1,54 @@
+exclude: ^lib/.*$
```

Comment:
Can you add a short introduction how to use this file in a comments section?

Reply:
It's in the git commit message, but I can add it there, too.

Reply:
Yes please. The commit message is out of sight if one finds this file.

Please also add a note that Python 3.7 is required. The best would be to proposal a patch upstream checking for min system requirements. 
Than we will get a good error measage instead of something obscure.